### PR TITLE
Implement "dir", fix "mv" and add oficial default direction when selecting character

### PR DESCRIPTION
**Pull request by Adanlink**

### CODE REVIEW

File: src/NosCore.Controllers/CharacterScreenPacketController.cs [hunk 1]
```diff
@@ -369,6 +369,7 @@ public void SelectCharacter(SelectPacket selectPacket)
                 character.MapInstance = _mapInstanceAccessService.GetMapInstance(character.MapInstanceId);
                 character.PositionX = character.MapX;
                 character.PositionY = character.MapY;
+                character.Direction = 2;
```
Comment (0Lucifer0) [inline src/NosCore.Controllers/CharacterScreenPacketController.cs]: Why 2 ? Should be last dir

File: src/NosCore.Controllers/DefaultPacketController.cs [hunk 2]
```diff
@@ -814,5 +814,27 @@ public void Rest(SitPacket sitpacket)
...
+        /// <param name="dirpacket"></param>
+        public void ChangeDir(ClientDirPacket dirpacket)
+        {
+            IAliveEntity entity;
+            switch (dirpacket.VisualType)
+            {
+                case VisualType.Player:
+                    entity = Session.Character;
+                    break;
+                //case VisualType.Monster:
```
Comment (0Lucifer0) [inline src/NosCore.Controllers/DefaultPacketController.cs]: No commented code

File: src/NosCore.Packets/ClientPackets/ClientDirPacket.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+using NosCore.Core.Serializing;
+using NosCore.Shared.Enumerations;
+using System.ComponentModel.DataAnnotations;
+
+namespace NosCore.Packets.ClientPackets
+{
+    [PacketHeader("dir")]
+    public class ClientDirPacket : PacketDefinition
+    {
+        #region Properties
```
Comment (0Lucifer0) [inline src/NosCore.Packets/ClientPackets/ClientDirPacket.cs]: No region

File: src/NosCore.Packets/ServerPackets/DirPacket.cs [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+using NosCore.Core.Serializing;
+using NosCore.Shared.Enumerations;
+
+namespace NosCore.Packets.ClientPackets
+{
+    [PacketHeader("dir")]
+    public class DirPacket : PacketDefinition
+    {
+        #region Properties
```
Comment (0Lucifer0) [inline src/NosCore.Packets/ServerPackets/DirPacket.cs]: No region

File: src/NosCore.Packets/ServerPackets/DirPacket.cs [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using NosCore.Core.Serializing;
+using NosCore.Shared.Enumerations;
+
+namespace NosCore.Packets.ClientPackets
```
Comment (0Lucifer0) [inline src/NosCore.Packets/ServerPackets/DirPacket.cs]: ServerPackets

Comment (Fizo55) [inline src/NosCore.Packets/ServerPackets/DirPacket.cs]: namespace NosCore.Packets.ServerPackets

Comment (Adanlink) [reply src/NosCore.Packets/ServerPackets/DirPacket.cs]: it is now good?

File: src/NosCore.Controllers/CharacterScreenPacketController.cs (hunk 1 above)
Comment (Adanlink) [reply src/NosCore.Controllers/CharacterScreenPacketController.cs]: Last dir? Dir is saved too? Anyways, official doesn't save the dir when logging out.

File: src/NosCore.Packets/ServerPackets/DirPacket.cs (hunk 5 above)
Comment (Adanlink) [reply src/NosCore.Packets/ServerPackets/DirPacket.cs]: Done 15 minutes ago

File: src/NosCore.Controllers/DefaultPacketController.cs (hunk 2 above)
Comment (Adanlink) [reply src/NosCore.Controllers/DefaultPacketController.cs]: Removed

File: src/NosCore.Packets/ClientPackets/ClientDirPacket.cs (hunk 3 above)
Comment (Adanlink) [reply src/NosCore.Packets/ClientPackets/ClientDirPacket.cs]: Removed

File: src/NosCore.Packets/ServerPackets/DirPacket.cs (hunk 4 above)
Comment (Adanlink) [reply src/NosCore.Packets/ServerPackets/DirPacket.cs]: Removed

File: src/NosCore.Controllers/CharacterScreenPacketController.cs (hunk 1 above)
Comment (0Lucifer0) [reply src/NosCore.Controllers/CharacterScreenPacketController.cs]: are you sure about that ?

Comment (Adanlink) [reply src/NosCore.Controllers/CharacterScreenPacketController.cs]: I checked it some hours ago, only to make sure I will go and check it now.

Comment (Adanlink) [reply src/NosCore.Controllers/CharacterScreenPacketController.cs]: Yeah, it's by default 2

**Pull request merged**